bears/hypertext/HTMLHintBear.py

```diff
@@ -0,0 +1,180 @@
+                        allow_attribute_value_in_double_quotes: bool=False,
+                        prohibit_empty_attribute: bool=False,
+                        prohibit_attribute_duplication: bool=True,
+                        use_doctype_at_beginning: bool=True,
+                        enforce_tag_pair: bool=True,
+                        enforce_self_close_empty_tag: bool=True,
+                        allow_escaped_special_characters: bool=False,
+                        enforce_unique_attribute_id: bool=True,
+                        require_title_tag: bool=True,
+                        disallow_script_in_head: bool=False,
+                        require_alt_attribute: bool=True,
+                        enforce_id_class_naming_convention: str='false',
+                        disallow_inline_style: bool=True,
+                        enforce_relative_links_in_href: bool=None,
+                        prohibit_unsafe_characters: bool=True,
+                        disallow_inline_script: bool = False,
+                        prohibit_style_tag: bool=False,
+                        htmlhint_config: str=''):
+        """
+        :param enforce_lowercase_tagname:
+            Enforce the tagnames to be written in lowercase.
+            For example: If set to ``True``, prefer ``<span><div>`` over
+            ``<SPAN><BR>``.
+        :param enforce_lowercase_attribute:
+            Enforce the attribute names to be written in lowercase.
+            For example: If set to ``True``, prefer
+            ``<img src="test.png" alt="test">`` over
+            ``<img SRC="test.png" ALT="test">``.
+        :param allow_attribute_value_in_double_quotes:
+            Allow attribute values to be enclosed in double quotes.
+            For example: If set to ``True``, prefer ``<a href="" title="abc">``
+            over ``<a href='' title=abc>``.
+        :param prohibit_empty_attribute:
+            Disallow empty values for attributes.
+            For example: If set to ``True``, prefer
+            ``<input type="button" disabled="disabled">`` over
+            ``<input type="button" disabled>``.
+        :param prohibit_attribute_duplication:
+            Disallow the defining of the same attribute more than once in
+            a tag. For example: If set to ``True``, prefer
+            ``<img src="a.png" />`` over ``<img src="a.png" src="b.png" />``.
+        :param use_doctype_at_beginning:
+            Enforce the ``<!DOCTYPE>`` declaration at the beginning.
+            For example: If set to ``True``, prefer ``<!DOCTYPE HTML><html>``
+            over ``<!--comment--><!DOCTYPE HTML><html>``.
+        :param enforce_tag_pair:
+            Enforce the tags to be paired.
+        :param enforce_self_close_empty_tag:
+            Enforce the empty tag to be closed by self.
+            For example: If set to ``True``, prefer ``<br />`` over ``<br>``.
+        :param allow_escaped_special_characters:
```

Comment:
again: You don't allow this case together with other cases, so you need to rename this too `require_...`